### PR TITLE
Adding test for SRVKP-3135

### PR DESCRIPTION
**Pull request by VeereshAradhya**

### CODE REVIEW

File: specs/operator/auto-prune.spec [hunk 1]
```diff
@@ -412,4 +412,34 @@ Steps:
   * Assert pruner cronjob(s) in namespace "target namespace" contains "1" number of container(s)
   * Delete project "test-project-1"
   * Delete project "test-project-2"
-  * Remove auto pruner configuration from config CR
\ No newline at end of file
+  * Remove auto pruner configuration from config CR
+
+## Verify that the operator is up and running after deleting namespace with pruner annotation: PIPELINES-12-TC15
+Tags: e2e, integration, auto-prune, admin, cronjob, sanity
```
Comment (ppitonak) [inline specs/operator/auto-prune.spec:418]: We have too many scenarios with `sanity` tag in this spec... we should limit their number significantly. I suggest to have just 1 or 2, e.g. the most basic one and the most complex one.

File: specs/operator/auto-prune.spec [hunk 2]
```diff
@@ -412,4 +412,34 @@ Steps:
...
+  * Remove auto pruner configuration from config CR
+  * Assert if cronjob with prefix "tekton-resource-pruner" is "not present" in target namespace
+  * Create
+      |S.NO|resource_dir                                 |
+      |----|---------------------------------------------|
+      |1   |testdata/pruner/namespaces/namespace-one.yaml|
+  * Assert if cronjob with prefix "tekton-resource-pruner" is "present" in target namespace
+  * Delete project "namespace-one"
+  * Sleep for "5" seconds
+  * Assert if cronjob with prefix "tekton-resource-pruner" is "not present" in target namespace
```
Comment (ppitonak) [inline specs/operator/auto-prune.spec:436]: `target namespace` means openshift-pipelines?

Comment (VeereshAradhya) [reply specs/operator/auto-prune.spec:436]: Yes.

File: specs/operator/auto-prune.spec (hunk 1 above)
Comment (VeereshAradhya) [reply specs/operator/auto-prune.spec:418]: Will remove it from few tests after testing this particular test case

Comment (ppitonak) [reply specs/operator/auto-prune.spec:418]: :+1:

**Review by ppitonak — APPROVED** [review]

**Pull request merged**